docs/tailwind.js

```diff
@@ -1,8 +1,6 @@
 var config = require('../lib/index').defaultConfig()
 
-config.colors = {
-  ...config.colors,
-
+config.colors = Object.assign(config.colors, {
```

Comment:
`Object.assign` mutates the first object in the parameter list, so it isn't technically required to assign this back to the property. I can't decide if I think it's harmless or misleading.

If you want to do this in an immutable way, you can rewrite it like:

```
config.colors = Object.assign({}, config.colors, { ... })
```

Note the extra empty anonymous object as the first param.

Reply:
Also this is just the docs config file so who fucking cares.